WordPress/gutenberg
# Font Library Modal: Group font variations as a list

Similar to #58834

## What?

This PR groups fonts as a list in the following two screens:

Font Library Modal > Library Tab > Select a font

![library](https://github.com/user-attachments/assets/b3f182c6-5415-4806-b07d-50a43d29e1e8)

Font Library Modal > Install Fonts Tab > Select a font

![install-fonts](https://github.com/user-attachments/assets/10e0b6e4-085a-4d75-8dce-9a45200a5625)


## Why?

The current font variation screen only displays a list of button elements, so screen reader users have no way of knowing how many fonts are displayed.

## How?

Similar to #58834, wrap each font in a list item element and wrap those in a list element.

## Testing Instructions

This PR is just a markup change and should not change anything in behavior or appearance. If you want to check the differences between files, [this split view](https://github.com/WordPress/gutenberg/pull/64029/files?diff=split&w=1) will be easy to understand.

### Testing Instructions for Keyboard

- Launch a screen reader such as NVDA or VoiceOver.
- Open the font library modal just by using the keyboard.
  -  Go to Library Tab > Select a font
  - Install Fonts Tab > Select a font
- When you focus on the beginning of the font list: You should hear something like this: `list  with 2 items  ABeeZee Normal  check box  not checked`
